docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:centos7.9.2009
+FROM quay.io/centos/centos:stream8
```

Comment:
curious if this is the official distribution

Reply:
I think it is. And Prestissimo is also using it.